docs/changelog.md

```diff
@@ -26,6 +26,10 @@ permalink: /changelog/
 * **feature-customtabs**
  * Sharing a URL from a custom tab always uses the current url of the session. [Bug 1831803](https://bugzilla.mozilla.org/show_bug.cgi?id=1831803)
 
+* **browser-toolbar**
```

Comment:
The comments from the other PR is still relevant here. Could you move this under the 124.0 section?